dist/game/index.d.ts

```diff
@@ -2,18 +2,22 @@
 /// <reference path="path-finder.d.ts" />
 /// <reference path="constants.d.ts" />
 /// <reference path="prototypes/index.d.ts" />
+/// <reference path="visual.d.ts" />
 
 declare module "game" {
   import { ERR_BUSY, ERR_INVALID_ARGS, ERR_NOT_ENOUGH_ENERGY } from "game/constants";
   import { Structure } from "game/prototypes";
 
-  export * as utils from "game/utils";
+  export * from "game/utils";
+
```

Comment:
Was the utils namespace removed? this does not seem to have anything to do with visuals though :)